deploy/chart/Chart.yaml

```diff
@@ -3,7 +3,7 @@ name: cloud-provider-equinix-metal
 description: Kubernetes Cloud Provider for Equinix Metal 
 type: application
 version: "0.1.0"
-appVersion: "master"
+appVersion: "main"
```

Comment:
Are there `helm upgrade` implications to this change?

Reply:
I don't believe so as it's my understanding that this helm doesn't actually exist anywhere other than this chart file. It's just an experiment.